mllib/src/test/scala/org/apache/spark/ml/feature/BucketizerSuite.scala

```diff
@@ -45,6 +46,9 @@ class BucketizerSuite extends SparkFunSuite with MLlibTestSparkContext {
       .setOutputCol("result")
       .setSplits(splits)
 
+    // copied model must have the same parent.
+    MLTestingUtils.checkCopy(bucketizer)
```

Comment:
This is OK for now since there is not an Estimator to produce a Bucketizer.  (There will be eventually.)